test/integration/targets/docker-registry/tasks/tests/docker_login.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Log in with wrong password (check mode)
+  docker_login:
+    registry_url: "{{ registry_frontend_address }}"
+    username: testuser
+    password: "1234"
+    state: present
+  register: login_failed_check
+  ignore_errors: yes
+
+- name: Log in with wrong password
+  docker_login:
+    registry_url: "{{ registry_frontend_address }}"
+    username: testuser
+    password: "1234"
+    state: present
+  register: login_failed
+  ignore_errors: yes
+
+- name: Make sure that login failed
+  assert:
+    that:
+      - login_failed_check is failed
+      - "('login attempt to http://' ~ registry_frontend_address ~ '/v2/ failed') in login_failed_check.msg"
+      - login_failed is failed
+      - "('login attempt to http://' ~ registry_frontend_address ~ '/v2/ failed') in login_failed.msg"
+
+#- name: Log in (check mode)
```

Comment:
Any reason to no detele these comments ? If there is an open issue/PR, mention the ID.

Reply:
It would be nice if the module could be properly implemented to no longer need these comments. There's no open issue right now (I think), but there's a big rewrite PR (#61207) which might already make this work (haven't checked it in detail yet). If it doesn't make this work, I'll create an issue.